lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -179,6 +179,11 @@ def compare_results(
                 {},
                 ["test", "fio_1", "fio_2", "uperf_1", "uperf_2", "uperf_3", "uperf_4"],
             ),
+            ("drb", {"owner": "drb"}, ["drb", "fio_1"]),
+            ("drb", {"mine": True}, ["drb", "fio_1"]),
+            ("drb", {"mine": "t"}, ["drb", "fio_1"]),
+            ("drb", {"mine": False}, ["fio_2"]),
+            ("drb", {"mine": "n"}, ["fio_2"]),
```

Comment:
No bad value testing?  (E.g., `mine` without authorization or `mine` together with `owner`?)

Reply:
Yeah, I had intended to do that, got distracted, and forgot. Darn. 😦